### ISSUE TITLE
Is there an SC for respecting user accessibility settings for cursor/pointer control?

### ISSUE THREAD

**Issue by caitgebs** [w3c/wcag#3161]
I have encountered a major accessibility issue on a site that I am auditing: https://www.borngroup.com/

This site uses a custom cursor style (i.e., small teal dot). This custom style overrides a user's Pointer Size, for example, an increased Large Size pointer. There are other examples in the wild, as detailed in this article: https://ericwbailey.website/published/dont-use-custom-css-mouse-cursors/

I have not been able to determine what SC this violates and curious if this will/is included in 2.2

Thank you!

**Comment by patrickhlauke** [comment]
In short, to answer your question: no, and no, I'm afraid.

**Comment by alastc** [comment]
Patrick is correct, but I'll label this wcag.next so it comes up when we next look at new guidelines.
